nss-day-cohort-43/nuttyshell-picnic-blanket
# Bl events ux

# Description
Changed a few small things to improve user experience with events.  The form to add a new event is now hidden and can be toggled by the user.  The weather for each event now displays more information.  The user should now only be able to either edit a single event or create a new event at one time.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
- Test displaying and hiding the event form
- Check that the weather displays correctly for each event
- Check that you can only create a new event or edit a single event (in other words, only one form for creating/submitting an event should display at a time)
- Test overall functionality of the events component/try and break it (minus using an invalid zip code)
# Checklist:
